ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -175,6 +178,8 @@ public void InvokeCallback(int callbackId, JArray arguments)
             IsDisposed = true;
 
             await QueueConfiguration.NativeModulesQueue.RunAsync(_registry.NotifyReactInstanceDispose).ConfigureAwait(false);
+            // RN Android is structured differently and notifies idle listeners here, should we do the same?
+            // CatalystInstanceImpl.java#L331
             await QueueConfiguration.JavaScriptQueue.RunAsync(() => _bridge?.Dispose()).ConfigureAwait(false);
```

Comment:
A long time ago, there was a TODO for this :-):
[https://github.com/Microsoft/react-native-windows/blame/1ef3fb438907545d2008ce51085afc5737e75f28/ReactNative/Bridge/CatalystInstance.cs#L102](https://github.com/Microsoft/react-native-windows/blame/1ef3fb438907545d2008ce51085afc5737e75f28/ReactNative/Bridge/CatalystInstance.cs#L102)

Reply:
better late than never ;) do you agree that the structure is different, so where we notify is appropriate given the divergence?

Reply:
I think this is just a convention that one final idle callback should occur when the `ReactInstance` is being torn down.

Reply:
#fixed